pyfluidsynth.py

```diff
@@ -107,16 +113,106 @@ def cfunc(name, result, *args):
+        ('iteration_start', c_void_p),
+        ('iteration_next', c_void_p)
+    ]
+
+try:
+    fluid_ramsfont_create_sfont = cfunc( 'fluid_ramsfont_create_sfont', POINTER(fluid_sfont_t) )
+
+    fluid_ramsfont_set_name = cfunc( 'fluid_ramsfont_set_name', c_int,
+                                ( 'sfont', c_void_p, 1 ),
+                                ( 'name', c_char_p, 1 ) )
+
+    fluid_ramsfont_add_izone = cfunc( 'fluid_ramsfont_add_izone', c_int,
+                                    ( 'sfont', c_void_p, 1 ),
+                                    ( 'bank', c_uint, 1 ),
+                                    ( 'num', c_uint, 1 ),
+                                    ( 'sample', c_void_p, 1 ),
+                                    ( 'lokey', c_int, 1 ),
+                                    ( 'hikey', c_int, 1 ) )
+
+    fluid_ramsfont_remove_izone = cfunc( 'fluid_ramsfont_remove_izone', c_int,
+                                    ( 'sfont', c_void_p, 1 ),
+                                    ( 'bank', c_uint, 1 ),
+                                    ( 'num', c_uint, 1 ),
+                                    ( 'sample', c_void_p, 1 ) )
+
+    fluid_ramsfont_izone_set_gen = cfunc( 'fluid_ramsfont_izone_set_gen', c_int,
+                                        ( 'sfont', c_void_p, 1 ),
+                                        ( 'bank', c_uint, 1 ),
+                                        ( 'num', c_uint, 1 ),
+                                        ( 'sample', c_void_p, 1 ),
+                                        ( 'gen_type', c_int, 1 ),
+                                        ( 'value', c_float, 1 ) )
+
+    fluid_ramsfont_izone_set_loop = cfunc( 'fluid_ramsfont_izone_set_loop', c_int,
+                                        ( 'sfont', c_void_p, 1 ),
+                                        ( 'bank', c_uint, 1 ),
+                                        ( 'num', c_uint, 1 ),
+                                        ( 'sample', c_void_p, 1 ),
+                                        ( 'on', c_int, 1 ),
+                                        ( 'loopstart', c_float, 1 ),
+                                        ( 'loopend', c_float, 1 ) )
+
+    new_fluid_ramsample = cfunc( 'new_fluid_ramsample', c_void_p )
+
+    delete_fluid_ramsample = cfunc( 'delete_fluid_ramsample', c_int,
+                                ( 'sample', c_void_p, 1 ) )
+
+    fluid_sample_set_name = cfunc( 'fluid_sample_set_name', c_int,
+                                ( 'sample', c_void_p, 1 ),
+                                ( 'name', c_char_p, 1 ) )
+
+    fluid_sample_set_sound_data = cfunc( 'fluid_sample_set_sound_data', c_int,
+                                    ( 'sample', c_void_p, 1 ),
+                                    ( 'data', c_void_p, 1 ),
+                                    ( 'nbframes', c_uint, 1 ),
+                                    ( 'copy_data', c_short, 1 ),
+                                    ( 'rootkey', c_int, 1 ) )
+except:
```

Comment:
Please avoid bare `except`:
* https://peps.python.org/pep-0008/#programming-recommendations
* https://docs.astral.sh/ruff/rules/bare-except/
* https://realpython.com/the-most-diabolical-python-antipattern/